packages/bot-web-ui/src/stores/dashboard-store.ts

```diff
@@ -120,8 +120,13 @@ export default class DashboardStore implements IDashboardStore {
     strategy_save_type = 'unsaved';
     active_tour_step_number = 0;
     show_toast = false;
+    toast_message = '';
```

Comment:
@farabi-deriv it is a .ts file, so please add type for a new variable in the interface IDashboardStore above

Reply:
done ✅